templates/core/index.html

```diff
@@ -1,817 +1,753 @@
-    <h3 class="u-sv3">Get Core. Get snaps. Get going.</h3>
-  </div>
-  <div class="row u-equal-height u-align--center u-vertically-center u-hide--medium u-hide--small">
-    <div class="col-3">
-      <div class="u-sv2">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/03aa6ccc-automotive_car-network.png",
-        alt="Automotive",
-        width="184",
-        height="107",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-        }}
       </div>
-    </div>
-    <div class="col-3">
-      <div class="u-sv2">
-        {{ image (
-          url="https://assets.ubuntu.com/v1/5e32acc5-Factory.svg",
-          alt="",
-          width="71",
-          height="60",
-          hi_def=True,
-          loading="lazy"
-          ) | safe
-        }}
+      <div class="p-equal-height-row__item">
+        <hr class="p-rule--highlight u-hide--medium u-hide--small is-muted" />
+        <h5 class="p-heading--5">Agile containerisation</h5>
       </div>
-    </div>
-    <div class="col-3">
-      <div class="u-sv2">
-        {{ image (
-          url="https://assets.ubuntu.com/v1/e39199bc-Internet+of+Things_digital-gateways.svg",
-          alt="Digital gateways",
-          width="80",
-          height="70",
-          hi_def=True,
-          loading="lazy"
-          ) | safe
-          }}
+      <p class="p-equal-height-row__item">
+        Ubuntu Core is immutable and strictly confined. There is a clean separation between the kernel, OS image and
+        applications, each updated independently and protected against corruption.
+      </p>
+      <div class="p-equal-height-row__item">
+        <p>
+          <a href="/core/docs/components#heading--confinement">Learn more about confinement&nbsp;&rsaquo;</a>
+        </p>
       </div>
     </div>
-    <div class="col-3">
-      <div class="u-sv2">
-        {{ image (
-          url="https://assets.ubuntu.com/v1/38efd13b-solution-robot.svg",
-          alt="",
-          width="51",
-          height="81",
-          hi_def=True,
-          loading="lazy"
-          ) | safe
-        }}
+    <div class="p-equal-height-row__col">
+      <div class="p-equal-height-row__item u-hide--small">
+        <div class="p-media-container">
+          <img width="320" src="https://assets.ubuntu.com/v1/166ec7c6-hardware-ecosystem.jpg" alt="" />
+        </div>
+      </div>
+      <div class="p-equal-height-row__item">
+        <hr class="p-rule--highlight u-hide--medium u-hide--small is-muted" />
+        <h5 class="p-heading--5">A strong hardware ecosystem</h5>
+      </div>
+      <p class="p-equal-height-row__item">
+        We enable Ubuntu Core with the best ODMs and silicon vendors in the world. We continuously test it on leading
+        IoT and edge devices and hardware.
+      </p>
+      <div class="p-equal-height-row__item">
+        <p>
+          <a href="/certified">Browse all certified hardware&nbsp;&rsaquo;</a> 
+        </p>
       </div>
```

Comment:
Missing [Discover the full feature list>](https://ubuntu.com/core/features) at the bottom of this section